Lib/blackrenderer/__init__.py

```diff
@@ -2,3 +2,10 @@
     from ._version import version as __version__
 except ImportError:
     __version__ = "<unknown>"
+
+# Optional command line arguments.
+class BlackRendererSettings:
```

Comment:
No, it should be in `__main__.py`, like I wrote.

Reply:
That breaks `flake8`

Reply:
Compare https://github.com/BlackFoundryCom/black-renderer/pull/69/commits/f4871c924538ba06d9c01b9e209c3280578a4cff to https://github.com/BlackFoundryCom/black-renderer/pull/69/commits/dd9f4c9a5c6064d1249111209251057d9c239e9e

Reply:
Ah, yes, I see. Let's put it in render.py then, that's perhaps the more logical place.